py-polars/tests/unit/io/test_lazy_json.py

```diff
@@ -100,3 +100,32 @@ def test_glob_n_rows(io_files_path: Path) -> None:
 # See #10661.
 def test_json_no_unicode_truncate() -> None:
     assert pl.read_ndjson(rb'{"field": "\ufffd1234"}')[0, 0] == "\ufffd1234"
+
+
+def test_sink_json_should_write_same_data(io_files_path: Path, tmp_path: Path) -> None:
+    tmp_path.mkdir(exist_ok=True)
+    # Arrange
+    source_path = io_files_path / "foods1.csv"
+    target_path = tmp_path / "foods_test.ndjson"
+    expected = pl.read_csv(source_path)
+    lz_df = pl.scan_csv(source_path)
+    # Act
+    lz_df.sink_json(target_path)
+    df = pl.read_ndjson(target_path)
+    # Assert
+    assert_frame_equal(df, expected)
+
+
+def test_sink_json_should_support_with_options(io_files_path: Path, tmp_path: Path) -> None:
```

Comment:
Nothing is being tested in this method?

Reply:
Sorry, we are still finishing the testing 😄